native/go/Makefile

```diff
@@ -19,8 +19,10 @@ include ../../mk/spksrc.native-cc.mk
 
 myComp:
 	@$(MSG) "Building Go"
-	cd $(WORK_DIR)/$(PKG_NAME)/src && ./make.bash
+	cd $(WORK_DIR)/$(PKG_NAME)/src && GOROOT_BOOTSTRAP=$(WORK_DIR)/../../go-1.4/work-native/go ./make.bash
 	@$(MSG) "Building Go for arm"
-	cd $(WORK_DIR)/$(PKG_NAME)/src && export GOARCH=arm && export GOARM=5 && ./make.bash --no-clean
+	cd $(WORK_DIR)/$(PKG_NAME)/src && export GOARCH=arm && export GOARM=5 && GOROOT_BOOTSTRAP=$(WORK_DIR)/../../go-1.4/work-native/go ./make.bash --no-clean
 	@$(MSG) "Building Go for 386"
-	cd $(WORK_DIR)/$(PKG_NAME)/src && export GOARCH=386 && ./make.bash --no-clean
+	cd $(WORK_DIR)/$(PKG_NAME)/src && export GOARCH=386 && GOROOT_BOOTSTRAP=$(WORK_DIR)/../../go-1.4/work-native/go ./make.bash --no-clean
+	@$(MSG) "Building Go for amd64"
```

Comment:
...see the README for supported environments? ;)
Seriously though, if that's the case, then probably hardcoding is the way to go. Explicitly do amd64 as the first one (which should build 386 first on your env), then 386, then arm? It removes one unnecessary 'rebuild' (fwiw, because it's more of a verification at that point) I guess.